caleb-lab/super-encode-me.js

```diff
@@ -0,0 +1,37 @@
+'use strict'
+
+const Tree = require('./models/tree.js')
+const Promise = require('bluebird')
+const fs = Promise.promisifyAll(require('fs'), {suffix: 'Prom'})
+let htmlTree = new Tree()
+// module.exports = function(router){
+//
+// }
+
+// let normalDocument = () => {
+fs.readFile(`${__dirname}/assets/minimal.html`, 'utf-8', function(err, data){
```

Comment:
You promisified the fs module, so why not use `.then() / .catch()` on the `readFile` method?